### PR TITLE
feat(autoware_launch): add predicted_path_checker package

### PR DESCRIPTION
**Pull request by brkay54**
## Description

Related links:
https://github.com/autowarefoundation/autoware.universe/pull/2528
https://github.com/autowarefoundation/autoware.universe/pull/3911
## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by brkay54** [comment]
Hi @TakaHoribe, the package was merged. Can you merge it?

File: autoware_launch/launch/components/tier4_control_component.launch.xml [hunk 1]
```diff
@@ -42,5 +43,7 @@
     <arg name="external_cmd_selector_param_path" value="$(find-pkg-share autoware_launch)/config/control/external_cmd_selector/external_cmd_selector.param.yaml"/>
     <arg name="aeb_param_path" value="$(find-pkg-share autoware_launch)/config/control/autonomous_emergency_braking/autonomous_emergency_braking.param.yaml"/>
     <arg name="enable_autonomous_emergency_braking" value="$(var enable_autonomous_emergency_braking)"/>
+    <arg name="enable_predicted_path_checker" value="$(var enable_predicted_path_checker)"/>
+    <arg name="predicted_path_checker_param_path" value="$(find-pkg-share autoware_launch)/config/control/predicted_path_checker/predicted_path_checker.param.yaml"/>
```
Comment (xmfcx) [inline autoware_launch/launch/components/tier4_control_component.launch.xml:47]: You are sending this `enable_predicted_path_checker` argument to the `control.launch.py` but you've forgotten to add it as a launch argument in the [previous PR](https://github.com/autowarefoundation/autoware.universe/pull/2528/files#r1342120806):

https://github.com/brkay54/autoware.universe/blob/1f2f0aaf82f103bef748e54448d2b2bf1fb93eef/launch/tier4_control_launch/launch/control.launch.py#L407

**Review by xmfcx — CHANGES_REQUESTED** [review]
You need to open up another PR in the universe repos to add the required launch argument.

File: autoware_launch/launch/components/tier4_control_component.launch.xml (hunk 1 above)
Comment (xmfcx) [reply autoware_launch/launch/components/tier4_control_component.launch.xml:47]: https://github.com/autowarefoundation/autoware.universe/pull/5186 is merged.

**Review by xmfcx — APPROVED** [review]

**Comment by TakaHoribe** [comment]
@brkay54 @xmfcx Thank you! I checked the simulator works too.

**Pull request merged**